polkadot-protocol/architecture/polkadot-chain/agile-coretime.md

```diff
@@ -1,4 +1,48 @@
 ---
 title: Agile Coretime
-description: TODO
----
\ No newline at end of file
+description: Explore the efficient scheduling mechanisms to access Polkadot cores to produce blockspace continuously or on-demand.
+---
+
+# Agile Coretime
+
+Agile Coretime is the [scheduling](https://en.wikipedia.org/wiki/Scheduling_(computing)){target=\_blank} 
```

Comment:
just to complement and make the learning experience even better, I was thinking about adding a small mermaid diagram here... 

```mermaid
flowchart TD
    A[Cores Designation] --> B[Bulk Coretime]
    A --> C[On-Demand Coretime]
 ```
@dawnkelly09, your input would be valuable here. However, this is not a priority, just a nice to have so if we are rushing feel free to skip this comment

Reply:
I added a Jira ticket for this. If we find time I'll get it in for launch. Otherwise, after.